sklearn/datasets/_openml.py

```diff
@@ -674,10 +674,15 @@ def fetch_openml(name=None, *, version='active', data_id=None, data_home=None,
         The Bunch will contain a ``frame`` attribute with the target and the
         data. If ``return_X_y`` is True, then ``(data, target)`` will be pandas
         DataFrames or Series as describe above.
+
         If as_frame is 'auto', the data and target will be converted to
         DataFrame or Series as if as_frame is set to True, unless the dataset
         is stored in sparse format.
 
+        .. versionchanged:: 0.25
```

Comment:
I am not sure what we want to do here in particular I am not sure we want to document something that will happen in the future (most likely not).

Reply:
This was actually suggested by @glemaitre. I think it makes sense to mention the planned change in docstring. If a user is reading docstring about this particular argument, it means the exact behavior matters for him/her. So I feel it's good to inform the user while they are at it. 
The alternative is we remove this `versionchange` tag and the description below all together. We would still have the warning message so it would be fine. What do you think?

Reply:
Maybe a reasonable compromise is to do something similar as this:

https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html
![image](https://user-images.githubusercontent.com/1680079/85693932-f6e9de80-b6d6-11ea-91d3-afeb467e8437.png)

So something like that:
```
.. versionchanged: 0.24
   The default value of strategy will change to "auto" in version 0.25. Starting from version 0.24,  
   a warning will be raised if ``as_frame`` is not explicitly set.
```

The key thing for me is that you are saying something about the current release (otherwise the doc says "Changed in version 0.25" although you are using the 0.24 release which is quite confusing. If I see something like this I immediately think this is a typo and 0.24 was meant rather than 0.25)

What do you think @glemaitre?

Reply:
Oh yes you are right. It should be 0.24

Reply:
That's a really good solution to this situation. Thanks!
Have updated the docstring accordingly.